src/ruby/PreProcessor.rb

```diff
@@ -25,7 +26,7 @@ def self.reset
   # Process the emerald to remove indentation and replace with brace convention
   # for an easier time parsing with context free grammar
   def self.process_emerald(file_name)
-    input = File.open(file_name, 'r').read
+    input = File.open(file_name, 'r').read.gsub('{', '\{')
```

Comment:
I think it's fine to leave in. Failing tests (in develop instead of master) are kind of like a to-do list of sorts but with an added incentive to fix later because of the red ci badge